CoroutineTemplate/README.md

```diff
@@ -40,3 +42,10 @@ $ ./gradlew jacocoTestReport
 ```
 
 Report is located at: `./app/build/reports/jacoco/`
+
+## Build and deploy
+
+For `release` builds, we need to provide release keystore and signing properties:
+
+- Put the `release.keystore` file at `app/config`.
```

Comment:
@luongvo Would it be better to have `config` folder on root level? 🙏

I think it will be cleaner to have _one_ `config` folder in the root level, instead of many scattered throughout the codebase

Reply:
@Tuubz it's okay, I agree. My previous thought is to put it at the `root` level but there is a config folder for checkstyle at `app` level so I put it here. Anw, we will remove checkstyle soon so let me move it back to the `root` config folder.